memspecs/JEDEC_256Mb_WIDEIO_SDR-200_128bit.xml

```diff
@@ -64,4 +64,8 @@
       <parameter id="vdd" type="double" value="1.8" />
       <parameter id="vdd2" type="double" value="1.2" />
   </mempowerspec>
+  <membankwiseparams>
```

Comment:
I think we should not put this to every file, only to the measured ones, what do you think? 
If this section is not present we use the default value.